docs/api.md

```diff
@@ -2143,6 +2151,39 @@ Identifies what kind of target this is. Can be `"page"`, `"service_worker"`, or
 #### target.url()
 - returns: <[string]>
 
+### class: Coverage
+
+#### coverage.startJSCoverage()
+- returns: <[Promise]> Promise that resolves when coverage is started
+
+#### coverage.stopJSCoverage()
+- returns: <[Promise]<[Array]<[Object]>>> Promise that resolves to the array of coverage reports for all non-anonymous scripts
+  - `url` <[string]> Script URL
+  - `text` <[string]> Script content
+  - `ranges` <[Array]<[Object]>> Parts of the script that were executed
```

Comment:
Are these ranges in order? Can ranges overlap?

Reply:
Ranges are sorted and non-overlapping. Added this to the description.